### PR TITLE
Moving CI from Jenkins to GitHub Actions

### PR DESCRIPTION
**Pull request by fatb38**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+          registry: ${{ secrets.REGISTRY }}
+          username: ${{ secrets.NEXUS_USERNAME }}
+          password: ${{ secrets.NEXUS_PASSWORD }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ secrets.REGISTRY }}/blip
+          tags: type=ref,enable=true,prefix=${{ steps.package-version.outputs.current-version }}-PR-,event=pr
```
Comment (ydrisR) [inline .github/workflows/ci.yml:123]: What if the event is not a PR? This means we can only publish on PR branches?

**Review by ydrisR — COMMENTED** [review]
a couple of questions

File: .github/workflows/ci.yml (hunk 1 above)
Comment (fatb38) [reply .github/workflows/ci.yml]: 👍🏻

**Review by ydrisR — APPROVED** [review]
LGTM

**Pull request merged**